src/data/read-through-data-cache.ts

```diff
@@ -315,6 +316,16 @@ export class ReadThroughDataCache implements ContiguousDataSource {
 
               this.log.info('Successfully cached data', { id, hash });
               try {
+                // Check if the ArNS name is preferred
```

Comment:
Factor this out into a method. Also, don't reference the config directly right here.